src/credentials/auth.ts

```diff
@@ -356,6 +356,12 @@ export class Auth implements AuthService, ConnectionManager {
         return connections.find(c => c.id === connection.id)
     }
 
+    /**
+     * Attempts to remove all auth state related to the connection.
+     *
+     * For SSO, this involves an API call to clear server-side state. The call happens
+     * before the local token(s) are cleared as they are needed in the request.
```

Comment:
[off-topic for this PR] it looks like provider.invalidate() is called even if client.logout() fails. Would it make sense to skip provider.invalidate() in that case?

Reply:
Maybe. It depends on user expectations + how the call failed. If we skip the local invalidation then users might not be directed to the browser the next time they select the profile. At least right now the call fails consistently for scoped tokens which is one reason to always clear.